packages/api-server/src/util.ts

```diff
@@ -41,7 +41,7 @@ export function snakeToCamel(
   let camel: any = {};
   Object.entries(t).map(([key, value]) => {
     let newValue =
-      typeof value === "object"
+      typeof value === "object" && value != null
         ? snakeToCamel(value, excludeKeys, depthLimit - 1)
```

Comment:
In my opinion, the better idea might be to handle type detection inside snakeToCamel, just [as the npm package "snake-camel" does](https://github.com/craftgear/snake-camel/blob/master/src/index.ts#L20-L22):

```ts
function camelToSnake(t, ..) {
  IF t is not type object {
    RETURN t;
  }

  ...
}
```

So that we don't have to check type of `t` outside of `snakeToCamel` and [here](https://github.com/godwokenrises/godwoken-web3/blob/c5206526dad33180d3568d2ede1a486898cf8066/packages/api-server/src/util.ts#L66).